### DISCUSSION TITLE
Displacement output

### DISCUSSION THREAD

**Discussion by Emmanuel-324** [idaholab/moose#28006] [Q&A Modules: Solid mechanics]
If I have this block below:
```
 [tdisp]
    type = FunctionDirichletBC
    variable = disp_x
    boundary = right
    function = '0.1*t'
  []
```
And I want to output it in my postprocessors block, what is the best way to do it.

I have already tried this below but I do not see it in my .out or csv file
```
 [tdisp]
    type = ElementAverageValue
    variable = disp_x
  []
```
Any suggestions?

**Comment by GiudGiud** [comment]
> I have already tried this below but I do not see it in my .out or csv file

A postprocessor certainly already shows up in your csv file. 
How did you create the csv output?

I think the best way to show displacements on a side would be this
https://mooseframework.inl.gov/source/vectorpostprocessors/SideValueSampler.html

**Reply by Emmanuel-324** [reply]
```
[table]
    type = CSV
    execute_on = timestep_end
    time_step_interval = 20
  []
```
